pliant/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,10 @@
+name: pliant
```

Comment:
Since the integration does not have code this file should be removed

Reply:
Hello,

The file has been removed.

Best,

The Pliant Team